src/data/FF7Text.cpp

```diff
@@ -53,96 +60,152 @@ QString FF7Text::toPC(QByteArray text)
         text.truncate(txt);
```

Comment:
Using var assignment and comparison on the if clause is a bad practice. You can here set txt variable on the line 51

```cpp
int endIndex = text.indexOf('\xFF');
if (endIndex != -1) {
    text.truncate(endIndex);
}
```

Reply:
Can Pause Wait or Memory have a 0xFF as a valid valid ?

Reply:
No longer truncate at the first 0xFF values can be 255  as well